src/main/java/net/dv8tion/jda/api/entities/Message.java

```
@@ -1179,6 +1181,311 @@ default boolean isFromGuild()
+     *      If the provided unicode emote is null or empty.
+     *
+     *  @return The {@link net.dv8tion.jda.api.requests.restaction.pagination.ReactionPaginationAction ReactionPaginationAction} of the emote's users.
+     */
+    @Nonnull
+    @CheckReturnValue
+    ReactionPaginationAction retrieveReactionUsers(String unicode);
+
+    /**
+     *  This obtains the {@link net.dv8tion.jda.api.entities.MessageReaction.ReactionEmote ReactionEmote} for the given reaction name on this message.
+     *
+     * <p>Messages can host a number of different reactions at once. In order to distinguish between reactions they are
+     * represented by their id or name. This method is specifically for referencing unicode emojis that follow the form ":name:"</p>
+     *
+     * @param name
+     *      The name of the reaction emoji.
+     *
+     * @throws java.lang.IllegalArgumentException
+     *      If the provided emote name is null or empty.
+     *
+     * @return The {@link net.dv8tion.jda.api.entities.MessageReaction.ReactionEmote ReactionEmote} of this message.
+     */
+    @Nullable
+    @CheckReturnValue
+    MessageReaction.ReactionEmote getReactionByName(String name);
+
+    /**
+     *  This obtains the {@link net.dv8tion.jda.api.entities.MessageReaction.ReactionEmote ReactionEmote} for the given reaction id on this message.
+     *
+     * <p>Messages can host a number of different reactions at once. In order to distinguish between reactions they are
+     * represented by their id or name. This method is specifically for referencing {@link net.dv8tion.jda.api.entities.Emote custom emotes}
+     * that follow the form "<:name:id>"</p>
+     *
+     * @param id
+     *      The string id of the reaction emoji.
+     *
+     * @throws java.lang.IllegalArgumentException
+     *      If the provided id is not a valid snowflake.
+     *
+     * @return The {@link net.dv8tion.jda.api.entities.MessageReaction.ReactionEmote ReactionEmote} of this message.
```

Or null?

---

Added "or null if not present" but I'm debating on if it should just throw an error instead